lib/charms/kratos/v0/kratos_info.py

```diff
@@ -104,7 +106,7 @@ def send_info_relation_data(
         info_databag = {
             "admin_endpoint": admin_endpoint,
             "public_endpoint": public_endpoint,
-            "login_browser_endpoint": f"{public_endpoint}/self-service/login/browser",
+            "login_browser_endpoint": urljoin(external_url, "/self-service/login/browser"),
```

Comment:
I am not sure if this will work, as usuall `external_url` includes a path:
```
>>> urljoin("https://traefik/model_charm", "/self-service/login/browser"),
('https://traefik/self-service/login/browser',)
```

IMO we should use `os.path.join` or parse the `external_url`